app/retail/templates/emails/quarterly_stats.html

```diff
@@ -0,0 +1,263 @@
+  }
+
+  .stat-contents {
+    color: #0D0764;
+    text-align: left;
+    width: 85%;
+    height: 100px;
+    align-items: center;
+    display: flex;
+    justify-content: left;
+  }
+
+  .stat-contents > p {
+    color: #0D0764;
+    padding: 1em;
+  }
+
+  .bounty-img {
+    border-radius: 50%;
+    background-color: #DBD0FF;
+    color: #3E00FF;
+    float: left;
+    width: 5em;
+    height: 5em;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .bounty-fund-img {
+    border-radius: 50%;
+    background-color: #D6FBEB;
+    color: #00A55E;
+    float: right;
+    width: 5em;
+    height: 5em;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: -3em;
+    margin-left: -1.3em;
+
+  }
+
+  .stat {
+    display: flex;
+    padding: 0.5em;
+    align-items: center;
+    justify-content: center;
+  }
+
+  #grow-oss {
+    margin-top: 5em;
+  }
+
+  .button-action {
+    margin-bottom: 5em;
+    margin-top: 3em;
+  }
+
+  .percent-content {
+    display: flex;
+    justify-content: center;
+  }
+
+  .button-action > .button {
+    padding: 1.3em;
+  }
+
+  .full-bordered-circle {
+    border-radius: 100%;
+    border-style: solid;
+    border-width: 5px;
+    border-color: #eeeeee;
+  }
+</style>
+<div class="stat-container">
+    <h1>GITCOIN QUARTERLY</h1>
+
+    {% if user_active_in_last_quarter %}
+        <h2>You've Been BUILDing</h2>
+        <h4>An overview of your stats from the last 3 months.</h4>
+        <div class="centered-contents stat-box">
+            <div class="stat-img total-transaction">
+                <p class="content">{{user_total_earned_usd}} USD</p>
+            </div>
+            <div class="stat-contents">
+                <p>You earned {{user_total_earned_eth}} ETH ({{user_total_earned_usd}} USD) from {{user_fulfilled_bounties_count}} completed bounties, for an average of {{user_avg_eth_earned_per_bounty}} ETH ({{ user_avg_usd_earned_per_bounty}} USD) per bounty!</p>
+            </div>
+        </div>
+        <div class="centered-contents stat-box">
+            <div class="stat-img full-bordered-circle percent-content">
+                <p class="content">{{user_bounty_completion_percentage}} %</p>
+            </div>
+            <div class="stat-contents">
+                <p>You completed {{user_bounty_completion_percentage}}% of the {{user_num_completed_bounties}} bounties that you started - thanks for showing such diligence.</p>
+            </div>
+        </div>
+        <div class="centered-contents stat-box">
+            <div class="stat-img full-bordered-circle">
+                <i class="fas fa-seedling"></i>
+            </div>
+            <div class="stat-contents">
+                {% comment %}
+                TODO: Find where we can fetch the below info from.
+                {% endcomment %}
+                <p>You've boosted your skills in 4 different languages: HTML, CSS, Python, and JavaScript!</p>
```

Comment:
can you put these variables into the python layer as an hardcoded array and @mbeacom and i can fill them out with information from the backend?